ui/pages/unlock-page/README.mdx

```diff
@@ -21,7 +21,7 @@ ArgsTable doesn't work with SectionShape
 | -------------------------- | -------------------------------------------------------------------------- |
 | `history`                  | History router for redirect after action `object`                          |
 | `isUnlocked`               | If isUnlocked is true will redirect to most recent route in history `bool` |
-| `onRestore`                | onClick handler for "import using Secret Recovery Phrase" link `func`      |
+| `onRestore`                | onClick handler for "Forgot Password?" link `func`      |
```

Comment:
nit: Should be sentence case?
```suggestion
| `onRestore`                | onClick handler for "Forgot password?" link `func`      |
```

Reply:
I was quoting the text on the link here, I am waiting on the design update on the cases as mentioned above

Reply:
Cool. That makes sense! Was just checking